fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/DefaultPaymentPeriodsInOneYearCalculator.java

```diff
@@ -45,7 +49,9 @@ public Integer calculate(final PeriodFrequencyType repaymentFrequencyType) {
             case INVALID:
                 paymentPeriodsInOneYear = Integer.valueOf(0);
             break;
-            default:
+            case WHOLE_TERM:
+                LOG.error("TODO Implement repaymentFrequencyType for WHOLE_TERM");
```

Comment:
Can this whole term case be implemented? Or is it too complicated? Can we have an issue opened for it? How will the whole term case even work?